tests/functional/utils.py

```diff
@@ -13,11 +14,21 @@
 
 
 def run_ggshield(
-    *args: str, expected_code: int = 0, cwd: Optional[PathLike] = None
+    *args: str,
+    expected_code: int = 0,
+    cwd: Optional[PathLike] = None,
+    env: Dict = dict(),
```

Comment:
mutable default argument. use None and then in the code create the dictionnary if the argument is None.

Typing should be corrected (`Optional[Dict]`)

Same for `run_ggshield_scan`